ReleaseProcessAutomation/Extensions/SemanticVersionExtensions.cs

```diff
@@ -26,6 +26,36 @@ public static class SemanticVersionExtensions
 {
   private static readonly ILogger s_log = Log.ForContext(typeof(SemanticVersionExtensions));
 
+  public static IReadOnlyCollection<SemanticVersion> GetNextPossibleVersionsReleaseBranchFromDevelop (this SemanticVersion semanticVersion)
+  {
+    if (semanticVersion.Pre == PreReleaseStage.rc)
+    {
+      return ReleaseRcOrCurrent(semanticVersion);
+    }
+    return new[]
+           {
+               GetNextMajorWithAlpha(semanticVersion),
+               GetNextMajorWithBeta(semanticVersion),
+               GetNextMajorRelease(semanticVersion)
+           };
+  }
```

Comment:
I'm not 100% sure about the semantics of this method. When I do a non-rc branch, I should get the alpha/beta for the current rtm version (major.minor.0), not "major.0.0" and I think the last option GetNextMajorRelease is also not current. 

It's probably something that's a hold over from the open review of everything. Maybe we just need to talk this through?

Reply:
Yeah, quick live sessions seem to be in order